cli-commands/init/init.js

```diff
@@ -51,6 +53,18 @@ const createTestsFolder = () => {
 	}
 }
 
+const copyPackageJsonFile = (libraryDirectory) => {
+	console.log('===== Generating package.json =====');
+
+	if (fs.existsSync(packageJsonDestination)) {
+		throw new Error(`package.json already exists in the main directory. You've probably already initialized etherlime for this project.`);
```

Comment:
Let's not throw an error if there is package son but just skip the things below by returning.